content/chainguard/chainguard-images/videos/zerocve.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "How Chainguard Creates Container Images with Low-to-No CVEs"
+linktitle: "Why our images have Low-to-No CVEs"
+lead: ""
+description: "This video explains how Chainguard are able to create container images with low-to-no
```

Comment:
Think I'd go with "Chainguard is"

Reply:
I kind of hate utilize (prefer use) but that's a hobby horse thing, feel free to ignore me.

Reply:
Ha, blame Will, I copied it from YouTube (I wouldn't use a z instead of an s normally ;) )

Reply:
The is/are thing is interesting, I think it's different between USA and UK.